Package.swift

```diff
@@ -28,6 +28,9 @@ let package = Package(
         .iOS(.v13)
     ],
     products: [
+        .library(
+            name: "SwiftToolsSupportBasic",
```

Comment:
Actually, can we call this "TSCBasic" instead? Since this only vends a single target, it makes sense to name the product and target the same.